group/urls.py

```diff
@@ -3,5 +3,8 @@
 from . import views
 
 urlpatterns = [
-    path('', views.create_group)
+    path('', views.create_group),
+    path('<int:group_id>/user/', views.add_users),
```

Comment:
Hmm, this isn't a good way to write url paths, usually, with the id coming first